js/main.js

```diff
@@ -1,6 +1,73 @@
 const menuIcon = document.querySelector('.menuIcon');
 const nav = document.querySelector('.navList');
 const linkedWord = document.querySelectorAll('.navWord');
+const modal = document.querySelector('.modals');
+const button = document.querySelectorAll('.cardBtn');
+const projectButton = document.querySelector('.seeButton');
+
+const projectCollection = [
```

Comment:
- [ OPTIONAL] Please it will nice to adjust the margins around the language buttons, see image below. 

***Your implementation***
![locI](https://user-images.githubusercontent.com/65852621/137477085-6736c19c-7b8b-49de-83eb-d774457fd699.png)
***The design***
![locD](https://user-images.githubusercontent.com/65852621/137477178-67c3f761-4715-48ae-875c-315b3bd3354d.png)